menpo/math/decomposition.py

```diff
@@ -1,16 +1,82 @@
 from __future__ import division
 import numpy as np
+from scipy.sparse import issparse
 from .linalg import dot_inplace_right
 
 
+def cca(X, Y, inplace=False):
```

Comment:
This is added and not used anywhere else in this PR - intentional? I would rather it was a separate PR if possible. Is it related to GRMF?